db/bigtable_eth1.go

```diff
@@ -222,32 +223,43 @@ func (bigtable *Bigtable) CheckForGapsInBlocksTable(lookback int) (gapFound bool
 }
 
 func (bigtable *Bigtable) GetLastBlockInBlocksTable() (int, error) {
-
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 	defer cancel()
+	redisKey := bigtable.chainId + ":lastBlockInBlocksTable"
 
-	prefix := bigtable.chainId + ":"
-	lastBlock := 0
-	err := bigtable.tableBlocks.ReadRows(ctx, gcp_bigtable.PrefixRange(prefix), func(r gcp_bigtable.Row) bool {
-		c, err := strconv.Atoi(strings.Replace(r.Key(), prefix, "", 1))
+	res, err := bigtable.redisCache.Get(ctx, redisKey).Result()
 
-		if err != nil {
-			logger.Errorf("error parsing block number from key %v: %v", r.Key(), err)
-			return false
-		}
-		c = max_block_number - c
+	if err != nil {
+		// key is not yet set, get data from bigtable
+		// key will be set via the eth1indexer
```

Comment:
This comment became obsolete via the last commit. Please remove it.
Also applies here:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/e4fd81eed01e0ca45a6caaea63a9b62ced109053/db/bigtable_eth1.go#L309-310